art/attacks/evasion/hop_skip_jump.py

```diff
@@ -127,6 +127,16 @@ def generate(self, x: np.ndarray, y: Optional[np.ndarray] = None, **kwargs) -> n
         else:
             start = 0
 
+        # Get the mask
+        mask = self._get_mask(x, **kwargs)
+
+        # Process mask as for each example
+        if mask is not None:
+            if len(mask.shape) == len(x.shape):
+                mask = mask.astype(ART_NUMPY_DTYPE)
+            else:
+                mask = np.array([mask.astype(ART_NUMPY_DTYPE)] * x.shape[0])
```

Comment:
Can we include the mask processing in `_get_mask`?

Reply:
Solved.